tests/runtime/engine/runner.py

```diff
@@ -22,9 +23,8 @@
 ERROR_COLOR = '\033[91m'
 NO_COLOR = '\033[0m'
 
-# TODO(mmarchini) only add colors if terminal supports it
```

Comment:
I'd leave the TODO here since this situation (terminal without color support) is still not covered.

Reply:
Done. (Also fixed typo that I forgot to commit and that caused the tests to fail.)